gunicorn/workers/sync.py

```diff
@@ -126,8 +126,37 @@ def handle(self, listener, client, addr):
         req = None
         try:
             if self.cfg.is_ssl:
-                client = ssl.wrap_socket(client, server_side=True,
-                    **self.cfg.ssl_options)
+                # TODO: Make this reusable by gthread worker. Move to BaseWorker?
+                context = ssl.SSLContext(self.cfg.ssl_options['ssl_version'])
+                context.set_default_verify_paths()
+                ciphers = self.cfg.ssl_options['ciphers']
+                # set_ciphers() will raise an exception if we pass None (default value)
+                if ciphers is not None and isinstance(ciphers, str):
```

Comment:
`if isinstance(ciphers, str):` should work, right?

``` python
>>> isinstance(None, str)
False
```

Reply:
Good point!